cmd/download-plugin.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/chris-cmsoft/concom/internal/downloader"
+	"github.com/hashicorp/go-hclog"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+	"path"
+)
+
+func DownloadPluginCmd() *cobra.Command {
+	var agentCmd = &cobra.Command{
+		Use:   "download-plugin",
+		Short: "downloads plugins from OCI or URLs",
+		Run: func(cmd *cobra.Command, args []string) {
+			logger := hclog.New(&hclog.LoggerOptions{
+				Output: os.Stdout,
+				Level:  hclog.Debug,
+			})
+			downloadCmd := DownloadRunner{
+				logger: logger,
+			}
+			err := downloadCmd.Run(cmd, args)
+			if err != nil {
```

Comment:
Perhaps `RunE:` at the param level? Probably be a bit cleaner.

Reply:
I don't quite get what you mean ? Can you expend a bit ?

Reply:
`RunE` is a way to not use `log.fatal` if returning an error at any time it looks like: https://pkg.go.dev/github.com/spf13/cobra#section-readme - just a nitpik, no worries

Reply:
Done :) Looks cleaner now